rules/integrations/aws/privilege_escalation_root_login_without_mfa.toml

```
@@ -22,7 +22,58 @@ interval = "10m"
 language = "kuery"
 license = "Elastic License v2"
 name = "AWS Root Login Without MFA"
-note = """## Setup
+note = """## Triage and analysis
+
+### Investigating AWS Root Login Without MFA
+
+Multi-factor authentication (MFA) in AWS is a simple best practice that adds an extra layer of protection on top of your
+user name and password. With MFA enabled, when a user signs in to an AWS Management Console, they will be prompted for
+their user name and password (the first factor—what they know), as well as for an authentication code from their AWS MFA
+device (the second factor—what they have). Taken together, these multiple factors provide increased security for your
```

I think we can omit identifying what is a first and second factor -- this feels like it's missing some extra explanation that we probably don't need here:
```suggestion
their user name and password, as well as for an authentication code from their AWS MFA
device. Taken together, these multiple factors provide increased security for your
```